RELEASES.md

```diff
@@ -1,3 +1,106 @@
+Version 1.65.0 (2022-11-03)
+==========================
+
+Language
+--------
+- [Error on `as` casts of enums with `#[non_exhaustive]`](https://github.com/rust-lang/rust/pull/92744/)
```

Comment:
```suggestion
- [Error on `as` casts of enums with `#[non_exhaustive]` variants](https://github.com/rust-lang/rust/pull/92744/)
```

AFAICT the check is for enum variants that are non_exhaustive, not if the whole enum is non_exhaustive.